Projekty/Projekt1/Adrian_Stańdo__Mateusz_Stączek__Paweł_Wojciechowski/feature_engineering.ipynb

```diff
@@ -0,0 +1,1966 @@
+{
```

Comment:
<p>Warto się zastanowić w ogóle nad zasadnością imputacji - ja to robię w tej chwili.</p><p> </p><p>Wydaje się, że Country jako USA ma sens skoro jest przeważająco dużo usańczyków, Private też wygląda w miarę sensownie.</p><p> </p><p>EDIT: Dobra, widzę, że ten wykres to właśnie robi. I właśnie Occupation nie działa, to może warto dać unknown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>